src/components/DeckBuilderDryRunView/index.js

```diff
@@ -397,11 +421,12 @@ class DeckBuilderDryRunView extends React.Component {
                       <WikiLink id='N12' />, <WikiLink id='N14' />,{' '}
                       <WikiLink id='N22' />, <WikiLink id='N33' />,{' '}
                       <WikiLink id='N48' />, <WikiLink id='W10' />,{' '}
-                      <WikiLink id='W12' />, and <WikiLink id='W19' />{' '}
-                      abilities, RNG for <WikiLink id='S3' />,{' '}
-                      <WikiLink id='W9' /> and <WikiLink id='W16' /> as well as
-                      cards than cannot actually be played in the first turn
-                      (e.g. <WikiLink id='F4' />, <WikiLink id='W1' />
+                      <WikiLink id='W12' />, <WikiLink id='W8' /> and{' '}
+                      <WikiLink id='W19' /> abilities, RNG for{' '}
+                      <WikiLink id='S3' />, <WikiLink id='W9' /> and{' '}
+                      <WikiLink id='W16' /> as well as cards than cannot
+                      actually be played in the first turn (e.g.{' '}
+                      <WikiLink id='F4' />, <WikiLink id='W1' />
                       …).
```

Comment:
This is getting super messy. I think at some point it might be nice to have a proper list of all the things that are supported, and how exactly. But that’s for another day. :)